composer.json

```diff
@@ -198,7 +198,7 @@
             "@php -d date.timezone=UTC ./vendor/bin/phpunit -c tests-legacy/old-controllers.xml"
         ],
         "integration-behaviour-tests": [
-            "@php -d date.timezone=UTC ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml"
+            "@php -d date.timezone=UTC ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -vv"
```

Comment:
Not sure it's a good idea to add the verbose mode since Travis have a limited amount of logs :thinking:

Reply:
I will remove it before squashing commits